content/en/docs/setup/platform-setup/k3d/index.md

```diff
@@ -23,7 +23,7 @@ k3d makes it very easy to create single- and multi-node k3s clusters in docker,
 1.  Create a cluster and disable `Traefik` with the following command:
 
     {{< text bash >}}
-    $ k3d cluster create --api-port 6550 -p "9080:80@loadbalancer"  -p "9443:443@loadbalancer" --agents 2 --k3s-arg '--disable=traefik@server:*'
+    $ k3d cluster create --api-port 6550 -p '9080:80@loadbalancer'  -p '9443:443@loadbalancer' --agents 2 --k3s-arg '--disable=traefik@server:*'
```

Comment:
Looking at the preliminary site, this is not resolved by using single quotes. I believe it's the way @'s are used as special links through the text shortcode.

Reply:
Yes, I think the fix for this is to make the regex for processing `@` chars more specific here:
https://github.com/istio/istio.io/blob/2d14e2e08dc7dedefd619be80c6526a33d153405/src/ts/codeBlocks.ts#L189

The `/@(.*?)@/g` part should maybe make sure that the string between the `@` chars is not a quote.

Maybe something like this will work? `/@([^"']*?)@/g`

Reply:
Ya.. I'm discussing with Greg but I have the fix. There's two places that need changing.

Reply:
https://github.com/istio/istio.io/pull/13637 has the fix